docs-metadata/package.json

```diff
@@ -0,0 +1,54 @@
+{
+	"name": "docs-metadata",
+	"displayName": "Docs Metadata",
+	"description": "Simplifies editing metadata within docs.",
+	"version": "0.0.1",
+	"publisher": "docsmsft",
+	"engines": {
+		"vscode": "^1.22.0"
+	},
+	"categories": [
+		"Other"
+	],
+	"activationEvents": [
+		"onCommand:extension.extract",
+		"onCommand:extension.apply"
+	],
+	"main": "./out/extension.js",
+	"contributes": {
+		"commands": [
+			{
+				"command": "extension.extract",
+				"title": "Docs Metadata: Extract"
```

Comment:
@adunndevster Everything looks great!  One question: do you think the title should be tweaked to read **Docs: Metadata extract** and **Docs: Metadata apply** to remain consistent with the other commands in the command palette?  Not a big deal at all and maybe it's different by design.  

![image](https://user-images.githubusercontent.com/11825888/59052495-10a65280-8844-11e9-8937-5e9b1e596f86.png)

Reply:
@meganbradley - What do you think?

Reply:
I'll change it. Easy enough to change back if needed :).